src/App.vue

```diff
@@ -156,6 +157,17 @@ export default class App extends Mixins(DateMixin) {
 
     // listen for spinner show/hide events
     this.$root.$on('showSpinner', (flag = false) => { this.showSpinner = flag })
+
+    // in case user is already logged in,
+    // get and store keycloak roles
+    try {
+      const keycloakRoles = getKeycloakRoles()
+      this.setKeycloakRoles(keycloakRoles)
+      console.info('Got roles!') // eslint-disable-line no-console
+    } catch (error) {
+      // just log the error message
+      console.log(`Did not get roles (${error.message})`) // eslint-disable-line no-console
```

Comment:
I decided to just log a friendly-looking message here (ie, not the stack trace) since this will happen every time a non-logged-in user loads this app.